.circleci/config.yml

```diff
@@ -29,3 +32,11 @@ jobs:
 
       - run: yarn lint
       - run: yarn danger ci --failOnErrors
+
+workflows:
+  build:
+    jobs:
+      - build:
+          context:
+          - org-global
```

Comment:
nice!

are there other repos that now need this same change?

Reply:
Thanks for asking! I've gotten most of our other repos. I think there are a few PRs that I still need to clean up.